docs/breakpad_reporting.rst

```diff
@@ -156,39 +156,36 @@ development instance. There are a few options:
       $ make shell
       app@c392a11dbfec:/app$ python -m testlib.mini_poster --url URL
 
-2. Use an addon/webextension:
-
-   * **Firefox < 57:** https://addons.mozilla.org/en-US/firefox/addon/crash-me-now-simple/
-   * **Firefox >= 59 nightly/beta:** https://github.com/rhelmer/webext-experiment-crashme
```

Comment:
These don't work anymore and we're pretty far beyond when they did work, so it's better to just not mention them anymore.